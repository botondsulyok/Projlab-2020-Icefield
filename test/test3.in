field spawn 0
field dry1 3
field wet1 5 true 0
field wet2 3 true 2
field dry2 0
neighbours spawn dry1
neighbours dry1 wet1
neighbours wet1 wet2
neighbours wet2 dry2
entity ib1 icebear spawn
stats ib1
move ib1 dry1
stats ib1
move ib1 wet1
stats ib1
move ib1 wet2
stats ib1
move ib1 dry2
stats ib1