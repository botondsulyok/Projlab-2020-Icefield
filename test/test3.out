id: ib1
type: icebear
iceUnit: spawn
movebehaviour: amph0

id: ib1
type: icebear
iceUnit: dry1
movebehaviour: amph0

id: ib1
type: icebear
iceUnit: wet1
movebehaviour: amph0

id: ib1
type: icebear
iceUnit: wet2
movebehaviour: amph0

id: ib1
type: icebear
iceUnit: dry2
movebehaviour: amph0

